src/arena/Sprints.tsx

```diff
@@ -81,7 +86,7 @@ function Sprints(props: SprintProps) {
                             {}
                           );
                           if (response) {
-                            alert('Succesfully deleted.');
```

Comment:
Can you change this to a Toast Notification?

Reply:
Its a done deal